### PR TITLE
Separate articles and recommendations

### PR DESCRIPTION
**Pull request by jonaharagon**
todo:

- [x] Add VPN description to top of overview page
- [x] Add perfect forward secrecy info to email security page

### CODE REVIEW

**Comment by dngray** [comment]
I've done https://github.com/privacyguides/privacyguides.org/pull/1236 so that should probably be taken into consideration

File: docs/email.en.md [hunk 1]
```diff
@@ -422,54 +422,6 @@ Must not have any marketing which is irresponsible:
...
-
-Email metadata is stored in the [message header](https://en.wikipedia.org/wiki/Email#Message_header) of the email message.
-
-### Why can't email metadata be E2EE?
-
-Email metadata is crucial to the most basic functionality of email (where it came from, and where it has to go). E2EE was not built into the email protocols originally and is also optional, therefore, only the message content is protected.
-
-### How is my metadata protected?
-
-When emails travel between email providers an encrypted connection is negotiated using [Opportunistic TLS](https://en.wikipedia.org/wiki/Opportunistic_TLS). This protects the metadata from outside observers, but as it is not E2EE, server administrators can snoop on the metadata of an email.
```
Comment (dngray) [inline docs/email.en.md:471]: Was there some reason you wanted to delete this?

**Comment by dngray** [comment]
I decided to delete these two links and add it to a requirement in https://github.com/privacyguides/privacyguides.org/pull/1173/commits/d9c60ef9f8887f4b25cb6c8d68f6b0d6cfc317bc.

Currently we don't recommend any US based email providers, and I doubt we will while warrant-less collection is possible under ECPA's 180 day rule. Seems there was some effort in 2017 for reform, but that too hasn't yet occurred.

**Comment by dngray** [comment]
> Add VPN description to top of overview page

I added an intro to the top, because I want to get this merged so I can work on https://github.com/privacyguides/privacyguides.org/pull/1311

File: docs/basics/email-security.en.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Email Metadata Overview
+
+Email metadata is stored in the [message header](https://en.wikipedia.org/wiki/Email#Message_header) of the email message, and includes some visible headers that you may have seen such as: `To`, `From`, `Cc`, `Date`, `Subject`. There are also a number of hidden headers included by many email clients and providers that can reveal information about your account.
+
+Client software may use email metadata to show who a message is from and what time it was received. Servers may use it to determine where an email message must be sent, among [other purposes](https://en.wikipedia.org/wiki/Email#Message_header) which are not always transparent.
+
+### Who Can View Email Metadata?
+
+Email metadata is able to be seen by your email client software (or webmail) and any servers relaying the message from you to any recipients. Sometimes email servers will also use third-party services to protect against spam, which generally also have access to your messages.
```
Comment (jonaharagon) [inline docs/basics/email-security.en.md:36]:
```suggestion
Email metadata is protected from outside observers with [Opportunistic TLS](https://en.wikipedia.org/wiki/Opportunistic_TLS) protecting it from outside observers, but it is still able to be seen by your email client software (or webmail) and any servers relaying the message from you to any recipients including your email provider. Sometimes email servers will also use third-party services to protect against spam, which generally also have access to your messages.
```

File: docs/email.en.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonaharagon) [reply docs/email.en.md:471]: https://github.com/privacyguides/privacyguides.org/pull/1173/files#r884163649

File: docs/basics/email-security.en.md (hunk 2 above)
Comment (dngray) [reply docs/basics/email-security.en.md]: Yeah looks good to me.

File: docs/basics/vpn-overview.en.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+---
+title: VPN Overview
+icon: material/vpn
+---
+
+Virtual Private Networks are a way of extending the end of your network to exit somewhere else in the world. An ISP can see the flow of internet traffic entering and exiting your network termination device (ie. modem).
+
+Encryption protocols such as HTTPS are commonly used on the internet, so they may not be able to see exactly what you're posting or reading but they can get an idea of the [domains you request](/basics/dns.md/#why-shouldnt-i-use-encrypted-dns).
+
+A VPN can help as it can shift trust to a server somewhere else in the world. As a result, the ISP then only sees that you are connected to a VPN and nothing about the activity that you're passing into it.
```
Comment (jonaharagon) [inline docs/basics/vpn-overview.en.md:10]:
```suggestion
Typically, Internet Service Providers can see any internet traffic entering and exiting your network. Encryption protocols such as HTTPS are commonly used on the internet, so they may not be able to see *exactly* what you're posting or reading, but they can get a general idea of things like the [domains you request](/basics/dns.md/#why-shouldnt-i-use-encrypted-dns).

A VPN can help in this situation, as it allows you to shift trust to a server somewhere else in the world. As a result, your ISP only sees that you are connected to a VPN, but nothing about the activity that you're passing into it.
```

**Pull request merged**